platform-hub-api/app/controllers/kubernetes/tokens_controller.rb

```diff
@@ -0,0 +1,112 @@
+class Kubernetes::TokensController < ApiJsonController
+
+  before_action :kube_identity
+
+  # GET /:user_id
+  def index
+    authorize! :read, @identity
+    render json: Kubernetes::TokenService.from_identity(@identity)
```

Comment:
The purpose of this method was to extract and return array of tokens from kubernetes identity. Will change the name to be more specific.